static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -218,6 +231,22 @@ const Common = styled('div')<{
   &:nth-last-child(3) {
     border-bottom-left-radius: 3px;
   }
+
+  &:after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    z-index: 1;
+    height: 2px;
+    width: 100%;
+    background-color: ${p => {
+      if (p.isCurrent) {
+        return p.theme.purple300;
+      }
+      return 'transparent';
+    }};
```

Comment:
```suggestion
    background-color: ${p => (p.isCurrent ? p.theme.purple300 : 'transparent')};
```

Reply:
> Wait, there's no hover bar when I hover over the Timeline, but i'm not within 1 second of a console message. I want to see a bar between console messages to represent the cursor position inside the Timeline

Done!

- Merged master and resolved conflicts
- Changed the bottom line to work as a border bottom instead of a pseudo-element (this was due to some problems with anti-aliasing with the dark mode)

https://user-images.githubusercontent.com/39612839/187756986-1b1487df-7a6d-4519-bf02-dc7cd01c1ad3.mp4